src/test/scala/edu/ie3/simona/test/common/input/EmInputTestData.scala

```diff
@@ -77,7 +81,7 @@ trait EmInputTestData
     Quantities.getQuantity(0.95, PU),
   )
 
-  protected val householdStorageInput = new StorageInput(
+  protected val storageInput = new StorageInput(
```

Comment:
Why did you change this name? Also, if you rename this value, you should probably rename the id too.